crowdsec-docs/unversioned/console/am-i-under-attack.md

```diff
@@ -0,0 +1,49 @@
+# Am I Under Attack
+
+## Introduction
+
+The feature Am I Under Attack is the pinnacle of what CrowdSec offers in terms of real-time infrastructure monitoring.
+
+Based on alerts shared by Security Engines, CrowdSec will perform an __AI-powered analysis of the alerts__ raised by the organization.
+
+If an unusual surge of signals is detected, that may __indicate that a targeted attack__ is ongoing on the organization.
+
+If the situation occurs, multiple communication channels will be used to reach out owner and administrator members of the organization.
+
+*   An email will be sent (max 1 / day)
+*   In-app notifications will be sent to the Console accounts (max 1/hour)
+*   A warning banner will be displayed on the Console accounts.
+
+  
+
+These inputs will contain a link to the alert page of the Console to get a precise overview giving the possibility to assess the situation and quickly react accordingly.
+
+  
+
+## Pre-requisites 
+
+*   The feature is only available for __Enterprise plan users__.
```

Comment:
Could we link to pricing page like EG:

```
This is an [enterprise plan](https://www.crowdsec.net/pricing) feature
```

Reply:
It's not available yet, as the feature is not publicly advertised yet.
But sure we'll have to change it for the official release